### DISCUSSION TITLE
react-staffdirectory problem

### DISCUSSION THREAD

**Discussion by JesusSanMol** [pnp/sp-dev-fx-webparts#2496] [Q&A]
Hi, I want to thank you all your work.

I have deployed the application following instructions and I approved permissions at the tenant level, but when I try to use it on any site the search box appears but it does not allow me to write anything and a wheel stays in the results box.

![image](https://user-images.githubusercontent.com/87412012/160424355-28932f47-d2d5-4adf-bcfa-341c39f817b3.png)

**Comment by gretchunkim** [comment]
@JesusSanMol I bet this issue is related to graph API permission issue since the Staff Directory utilize Azure AD and not SP service. 
Try to log into Graph API and see you consented if needed.
